### ISSUE TITLE
How to fit a dress

### ISSUE THREAD

**Issue by Pbb-land** [liren2515/ISP#3]
Hello, thanks for your great work.

I found that you can get the mask of the dress from the shcp parsing result in the process_segmentation function. If I set the fabric type = 3, can this model fit a dress like the one in the picture below（with just single layer）?
![image](https://github.com/liren2515/ISP/assets/60537777/79108505-1c4a-45b9-a6f9-a979cd853e0e)

**Comment by liren2515** [comment]
No. Since we don't have a model for long dress, it is not possible for you to recover such a dress. But this is feasible if we train ISP on this type of cloth.
